tasks/mint.js

```diff
@@ -0,0 +1,39 @@
+const path = require('path');
+const yargs = require('yargs/yargs');
+const { hideBin } = require('yargs/helpers');
+const argv = yargs(hideBin(process.argv)).parserConfiguration({
+    'parse-numbers': false
+}).argv;
+
+const getTestTokenInstance = async (from) => {
+    const testTokenArtifact = require(path.resolve(__dirname, '../build/contracts/TestToken.json'));
+    const TruffleContract = require('@truffle/contract');
+    const TestToken = TruffleContract(testTokenArtifact);
+    TestToken.defaults({ from });
```

Comment:
We could specify it here, or on line#32.
```typescript
const tx = await testTokenInstance.mint(amount, tokenReceiver);
```

I preferred to specify it here, because that's actually what Truffle does under the hood when we run `truffle console`. If you think it hurts readability, or if for any other reason, we could change it and specify the parameter only when required.

Reply:
I think it doesn't apply anymore, after the changes proposed by you ;-)